components/todo-input.js

```diff
@@ -4,17 +4,86 @@ import styled from "styled-components";
 export default function TodoInput() {
   return (
     <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
+      <TimeInput>
+        <Time>시간</Time>
+        <TimeInputArea placeholder="날짜를 입력하세요 (ex.20200404)" />
+      </TimeInput>
+      <TaskInput>
+        <Todo>TODO</Todo>
+        <TaskTextArea placeholder="할 일을 입력하세요" />
+      </TaskInput>
+      <Submit>등록</Submit>
     </Wrapper>
   );
 }
+const Submit = styled.button`
+  color: white;
+  background-color: rgb(97, 97, 97);
+  font-size: 1.5rem;
+  outline: none;
+  border-width: initial;
+  border-style: none;
+  border-color: initial;
+  border-image: initial;
+  padding: 0.5rem 1rem;
+  border-radius: 0.3rem;
+`;
+const Time = styled.p`
+  font-size: 1.5rem;
+  padding: 0px;
+  margin: 0px;
+`;
+const TimeInputArea = styled.input`
+  width: 80%;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgb(97, 97, 97);
+  border-image: initial;
```

Comment:
이 부분은 아래와 같이 border속성을 통해서 한번에 정리할 수 있습니다.

```suggestion
  border:1px solid rgb(97, 97, 97);
```